### DISCUSSION TITLE
Add "Connect null values" property to sparkline on the Stat panel

### DISCUSSION THREAD

**Discussion by charlie-haley** [grafana/grafana#37164] [Dashboards, Panels & Visualizations]
Add "Connect null values" property to stat panel, that behaves in a similar way to the time series panel. 

I have some data that reports quite sporadically. I'd like to see connected values on the sparkline background on the stat panel, ex.
![image](https://user-images.githubusercontent.com/27928201/126805010-7c4ea23b-f45f-413c-b887-a0b1b99be8ef.png)

**Comment by ian-rose** [comment]
+1
This would be a very good option to have.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
